src/js/views/form.f7.jsx

```diff
@@ -1,62 +1,95 @@
-import formTextbox from "./formTextbox.f7.jsx";
-import formSelect from "./formSelect.f7.jsx";
+import FormButton from "./formButton.f7.jsx";
+import FormCheckbox from "./formCheckbox.f7.jsx";
+import FormConnect from "./formConnect.f7.jsx";
+import FormCustom from "./formCustom.f7.jsx";
+import FormDatepicker from "./formDatepicker.f7.jsx";
+import FormJSON from "./formJSON.f7.jsx";
+import FormNumber from "./formNumber.f7.jsx";
+import FormReadonly from "./formReadonly.f7.jsx";
+import FormSelectMultiple from "./formSelectMultiple.f7.jsx";
+import FormSelectSingle from "./formSelectSingle.f7.jsx";
+import FormTextbox from "./formTextbox.f7.jsx";
+import FormTree from "./formTree.f7.jsx";
 
 export default class F7ViewForm {
-   // constructor(dcID, allDCs, $, $f7, $store, record) {
-   constructor(settings, parent, AB) {
-      this.dcID = settings.dcID;
-      this.formID = settings.formID;
-      this.parent = parent;
-      this.Application = parent.Application;
-
-      this.AB = AB;
-      this.$f7 = AB.$f7;
-      this.$ = AB.$;
-      this.$store = AB.$store;
-      this.isLoading = false;
+   #AB;
+   #settings;
+   constructor(AB, settings) {
+      this.#AB = AB;
```

Comment:
why use `.#AB` instead `.AB`?
same for `.#settings`?

Reply:
These are private variables.